src/recordlinker/linking/mpi_service.py

```diff
@@ -57,6 +57,21 @@ def get_block_data(
     return session.execute(expr).scalars().all()
 
 
+def fetch_person_records(
```

Comment:
Alternatively, we could have modified [`get_block_data()`](https://github.com/CDCgov/RecordLinker/pull/97/files#diff-ae6794ac378c7dc01b44005272a2254eabd9bca9ef6ab4169f93834c2b34ddeeL17), specifically lines [25](https://github.com/CDCgov/RecordLinker/pull/97/files#diff-ae6794ac378c7dc01b44005272a2254eabd9bca9ef6ab4169f93834c2b34ddeeL25) and [56](https://github.com/CDCgov/RecordLinker/pull/97/files#diff-ae6794ac378c7dc01b44005272a2254eabd9bca9ef6ab4169f93834c2b34ddeeL56), but I decided not to in order to preserve the definition of what "block data" is; "block data" should in fact just be those candidate Patient records identified in Blocking. However, in order to calculate Belongingness Ratio, we want to fetch all the Patient records within the Person clusters identified in Blocking. Open to feedback here, but I think modifying [`get_block_data()`](https://github.com/CDCgov/RecordLinker/pull/97/files#diff-ae6794ac378c7dc01b44005272a2254eabd9bca9ef6ab4169f93834c2b34ddeeL17) would be misleading by returning Patient records that technically were _not_ blocked on.